content/projects/data-science-specific/cross-validation-and-simple-linear-regression/_index.md

```diff
@@ -35,62 +36,53 @@ title: Cross-validation & Simple Linear Regression
+   
+3. Split your data into a training and test set. Leave the test set for now. Examine the training data for missing and extreme values. Create different histograms to show the distribution of each variable in your dataset. Create a scatterplot showing the relationship between Years Worked and Salary. Are the data appropriate for linear regression? Is there anything that needs to be transformed or edited first?
+
+4. Using the ``statsmodels`` package and the _training data_, run a simple linear regression for Salary with one predictor variable: Years Worked.
    - Does the model significantly predict the dependent variable? Report the amount of variance explained (R^2) and significance value (p) to support your answer.
    - What percentage of the variance in employees’ salaries is accounted for by the number of years they have worked?
 
-3. What does the unstandardised coefficient (B or 'coef' in `statsmodels`) tell you about the relationship between Years Worked and Salary?
-
-4. What do the 95% confidence intervals [0.025, 0.975] mean?
-
-5. Calculate the expected salary for someone with 12 years’ work experience.
+5. What does the unstandardised coefficient (B or 'coef' in `statsmodels`) tell you about the relationship between Years Worked and Salary?
 
-6. Calculate the expected salary for someone with 80 years’ work experience. Are there any problems with this prediction? If so, what are they?
+6. What do the 95% confidence intervals [0.025, 0.975] mean?
 
-7. Now fit your model to your test set. DO NOT BUILD A NEW MODEL ON THE TEST SET! Simply use your existing, model, to predict salaries in the test set.
+7. Calculate the expected salary for someone with 12 years’ work experience.
 
-8. How does your model compare when running it on the test set - what is the difference in the Root Mean Square Error (RMSE) between the training and test sets? Is there any evidence of overfitting?
+8. Calculate the expected salary for someone with 80 years’ work experience. Are there any problems with this prediction? If so, what are they?
 
-9. We have only looked at the number of years an employee has worked. What other employee characteristics might influence their salary?
+9. Now fit your model to your test set. DO NOT BUILD A NEW MODEL ON THE TEST SET! Simply use your existing, model, to predict salaries in the test set.
 
-### References
+10. How does your model compare when running it on the test set? - what is the difference in the Root Mean Square Error (RMSE) between the training and test sets? Is there any evidence of overfitting?
 
-Data is made up and inspired by Cohen, Cohen, West & Aiken. Applied Multiple Regression/Correlation Analysis for the Behavioral Sciences, 3rd Edition.
+11. We have only looked at the number of years an employee has worked. What other employee characteristics and why do you think these characteristics might influence their salary?
```

Comment:
Maybe better to say

We have only looked at the number of years an employee has worked. What other employee characteristics might influence their salary? Please justify your answer using the data